### PR TITLE
feat:  calibrator tier4 pkg name

### PR DESCRIPTION
**Pull request by taikitanaka3**
## Description(required)

#141 

to tier4 calibration rviz plugin
```
Starting >>> tier4_control_msgs
Finished <<< tier4_control_msgs [4.45s]                     
Starting >>> tier4_vehicle_msgs
Finished <<< tier4_vehicle_msgs [7.65s]                      
Starting >>> tier4_calibration_rviz_plugin
Finished <<< tier4_calibration_rviz_plugin [9.47s]                      

Summary: 3 packages finished [22.1s]
```

## Review Procedure(required)

see build success

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [commit-guidelines][commit-guidelines]
- [ ] Assign PR to reviewer

If you are adding new package following items are required:

- [ ] Documentation with description of the package is available
- [ ] A sample launch file and parameter file are available if the package contains executable nodes

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[commit-guidelines]: https://www.conventionalcommits.org/en/v1.0.0/
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by tkimura4** [comment]
I think `tier4_calibration_rviz_plugin` should be in common directory.

File: vehicle/accel_brake_map_calibrator/tier4_calibration_rviz_plugin/CMakeLists.txt [hunk 1]
```diff
@@ -20,7 +20,7 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 add_definitions(-DQT_NO_KEYWORDS)
 
 ament_auto_add_library(${PROJECT_NAME} SHARED
-  src/accel_brake_map_calibrator_button_panel.cpp
+  src/tier4_calibration_rviz_plugin.cpp
```
Comment (tkimura4) [inline vehicle/accel_brake_map_calibrator/tier4_calibration_rviz_plugin/CMakeLists.txt:23]: The file name is better to be accel_brake_map_calibrator_button_panel.cpp.

Comment (taikitanaka3) [reply vehicle/accel_brake_map_calibrator/tier4_calibration_rviz_plugin/CMakeLists.txt]: fixed at 
https://github.com/autowarefoundation/autoware.universe/pull/200/commits/ec491580675aeefd8964e3fc152b8425ea2c04f9

**Comment by tkimura4** [comment]
Could you update package.xml/CmakeList?

**Review by 1222-takeshi — APPROVED** [review]
LGTM

**Pull request merged**